### ISSUE TITLE
Extension not working on OperaGX

### ISSUE THREAD

**Issue by alta1141** [laylavish/uBlockOrigin-HUGE-AI-Blocklist#64]
to my knowledge, operagx uses chromium like google chrome so i assumed the extension would work. added the url into the subscription + re-opened web browser as per instructions, but the ai images still appear on search
![image](https://github.com/user-attachments/assets/44ac4005-ea09-4cec-91c6-f556728e54ee)
![image](https://github.com/user-attachments/assets/9598c730-b24a-492d-9b97-b1ef855d78bb)

**Comment by StrongSand94191** [comment]
You shouldn't be using Opera GX in the first place: https://www.spacebar.news/stop-using-opera-browser/

**Comment by Mickmick21** [comment]
> Sorry, I know it doesn't solve the issue but you shouldn't be using Opera GX in the first place: https://www.spacebar.news/stop-using-opera-browser/

I agree, don't use Opera, i recommend switching to brave or chrome or something not Opera. Opera as been bought in 2016 by a chinese company and is collecting your data without you knowing or consenting to.

**Comment by StrongSand94191** [comment]
Use Vivaldi if you want a Chromium base or Librewolf if you want a Firefox base

**Comment by Stevoisiak** [comment]
> Opera as been bought in 2016 by a chinese company and is collecting your data without you knowing or consenting to.

Aren't most major browsers collecting user data? Not that there aren't privacy preserving alternatives, just unsure if Opera is worse than [Chrome](https://www.pcworld.com/article/2941324/3-ways-chrome-invades-your-privacy-and-what-you-can-do-about-it.html), [Firefox](https://www.zdnet.com/article/the-firefox-i-loved-is-gone-how-to-protect-your-privacy-on-it-now/), or [Microsoft Edge](https://windowsforum.com/threads/edge-privacy-guide-what-microsoft-collects-and-how-to-control-it.383383/) in terms of privacy.

**Comment by guest20** [comment]
Is your screenshot of google dot com, or a country specific google?

**Comment by Julmik647** [comment]
If you use ad blocking extension, let it acces search websites as it's blocked by defualt. I don't know if it's possibile with built-in ad blocker.

**Comment by Stevoisiak** [comment]
@Julmik647 Changing that setting worked in uBlock Origin for Opera.

<img width="740" height="160" alt="Image" src="https://github.com/user-attachments/assets/aa53fe40-0878-4479-b7f0-8f8a5d8e5554" />
